templates/partials/timestamp.html

```diff
@@ -1,10 +1,8 @@
-{{#unless containedIn}}
-	{{#with @nTeaserPresenter.timeObject}}
-		<div class="o-teaser__timestamp{{#if classModifier}} o-teaser__timestamp--{{classModifier}}{{/if}}">
-			{{#if status}}
-				<span class="o-teaser__timestamp-prefix">{{status}}</span>
-			{{/if}}
-			<time data-o-component="o-date" class="o-date" data-o-date-format="time-ago-limit-4-hours" datetime="{{publishedDate}}">{{#dateformat "dddd, d mmmm, yyyy"}}{{publishedDate}}{{/dateformat}}</time>
-		</div>
-	{{/with}}
-{{/unless}}
+{{#with @nTeaserPresenter.timeObject}}
+	<div class="o-teaser__timestamp{{#if classModifier}} o-teaser__timestamp--{{classModifier}}{{/if}}">
+		{{#if status}}
+			<span class="o-teaser__timestamp-prefix">LIVE</span>
+		{{/if}}
```

Comment:
This should be {{status}}, and not hardcoded to LIVE

Reply:
https://github.com/Financial-Times/n-teaser/pull/146/commits/9eb1f69fde40685f010093001bf87720a0b69ae6